cfme/utils/appliance/__init__.py

```diff
@@ -2963,6 +2963,14 @@ def create_view(self, view_class, o=None, override=None):
         return self.appliance.browser.create_view(
             view_class, additional_context={'object': new_obj})
 
+    def list_destinations(self):
```

Comment:
**REQUIRED:** Add a docstring.   In particular it took me a moment to discern what these were destinations to, so try to clarify that.